### DISCUSSION TITLE
pre-receive hook error message

### DISCUSSION THREAD

**Discussion by manish23jan2000** [community/community#58916] [Enterprise]
### Select Topic Area

General

### Body

I have added one script as pre-receive hook at enterprise level. It is behaving as per my expectation but problem is in case of uploading the file from gui. I am receiving error message as commit failed whereas i am expecting an error message which is mentioned in my pre-receive hook script.

Through CLI i am getting expected message.also while adding new file from gui i am getting expected error message.
Can some one help

**Comment by SUBHADIPMAITI-DEV** [comment]
Check the implementation of your pre-receive hook script: Double-check your script to ensure that it is correctly written and the error message you expect is included in the appropriate conditions.

Verify the hook script is properly set up: Confirm that the pre-receive hook script is properly set up at the enterprise level and is being triggered when committing changes through the GUI. Check the documentation or guidelines for the platform or version control system you are using to ensure you have correctly configured the hook script.

Review the error logs: Look for any error logs or console output that might provide additional information about why the specific error message from your hook script is not being displayed. Check the logs for any potential errors or conflicts that might be interfering with the execution of your script.

Test with different scenarios: Try different scenarios, such as uploading different types of files or making specific changes, to see if the issue occurs consistently or only under certain conditions. This can help you narrow down the problem and identify any patterns or specific situations where the error message is not displayed as expected.

Consult the documentation or support channels: If you have checked all the above steps and are still unable to resolve the issue, it may be helpful to consult the documentation or support channels specific to the platform or version control system you are using. They might be able to provide further guidance or identify any known issues related to pre-receive hook scripts and GUI uploads.
